Sources/PackageCollections/Storage/SQLitePackageCollectionsStorage.swift

```diff
@@ -798,6 +798,11 @@ final class SQLitePackageCollectionsStorage: PackageCollectionsStorage, Closable
         """
         try db.exec(query: table)
 
+        #if os(Android)
+        // FTS queries for strings containing hyphens isn't working in SQLite on
+        // Android, so disable for now.
+        useSearchIndices.put(false)
```

Comment:
Rebased and disabled FTS on Android, all package collections tests now pass on Android with this change and #3402.